adam-core/src/test/scala/org/bdgenomics/adam/rdd/ADAMContextSuite.scala

```diff
@@ -17,22 +17,21 @@
  */
 package org.bdgenomics.adam.rdd
 
-import htsjdk.samtools.{
-  SAMFormatException,
-  SAMProgramRecord,
-  ValidationStringency
-}
+import htsjdk.samtools.{ SAMFormatException, SAMProgramRecord, ValidationStringency }
 import java.io.{ File, FileNotFoundException }
+
 import com.google.common.io.Files
 import org.apache.hadoop.fs.Path
 import org.apache.parquet.filter2.dsl.Dsl._
 import org.apache.parquet.filter2.predicate.FilterPredicate
 import org.apache.parquet.hadoop.metadata.CompressionCodecName
 import org.apache.spark.rdd.RDD
+import org.apache.spark.sql.SQLContext
 import org.bdgenomics.adam.models._
 import org.bdgenomics.adam.rdd.ADAMContext._
 import org.bdgenomics.adam.util.PhredUtils._
 import org.bdgenomics.adam.util.ADAMFunSuite
+import org.bdgenomics.adam.sql.{ VariantContext => VCProduct }
```

Comment:
Code style: don't add unnecessary abbreviations, especially in class names

Reply:
+1